src/measurements.ts

```diff
@@ -1,80 +1,186 @@
 import * as THREE from "three";
 import { Utils } from "./utils";
 
+interface SpherePair {
+  sphere1: THREE.Mesh<THREE.SphereGeometry, THREE.MeshBasicMaterial>;
+  sphere2: THREE.Mesh<THREE.SphereGeometry, THREE.MeshBasicMaterial>;
+}
```

Comment:
```suggestion
}

private static readonly METERS_TO_INCHES = 39.3701;
```

Can we try this. Removing the initialization in the class and removing `this` when calling the variable